components/server/ee/src/prebuilds/gitlab-service.ts

```diff
@@ -32,7 +32,7 @@ export class GitlabService extends RepositoryService {
             return false;
         }
         const api = await this.api.create(user);
-        const response = (await api.Projects.show(`${owner}/${repoName}`)) as unknown as GitLab.Project;
+        const response = (await api.Projects.show(`${owner}%2F${repoName}`)) as unknown as GitLab.Project;
```

Comment:
Note: I believe this is not necessary, as `api.Projects.show` already URIencodes this:

```js
// workspace/gitpod/node_modules/@gitbeaker/core/dist/index.js:2061
    Projects.prototype.show = function (projectId, options) {
        var pId = encodeURIComponent(projectId);
        return RequestHelper.get(this, "projects/" + pId, options);
    };
```